src/python/pants/backend/python/rules/coverage_plugin/plugin.py

```diff
@@ -3,38 +3,41 @@
 
 import json
 import os
-import sys
 
-from coverage import CoveragePlugin, FileTracer
+from coverage import CoveragePlugin
 from coverage.config import DEFAULT_PARTIAL, DEFAULT_PARTIAL_ALWAYS
 from coverage.misc import join_regex
 from coverage.parser import PythonParser
 from coverage.python import PythonFileReporter, get_python_source
 
 
-# Note: This plugin will appear to do nothing at all at test time. This is the correct behavior. The *only* thing
```

Comment:
Solely reformatted + some capitalization changes. Ditto for all comments in this file.